sdk/include/reactos/wine/port.h

```diff
@@ -351,7 +351,7 @@ extern int spawnvp(int mode, const char *cmdname, const char * const argv[]);
 
 #endif  /* __i386___ && __GNUC__ */
 
-#if defined(_MSC_VER)
+#if defined(_MSC_VER) && !defined(__clang__)
```

Comment:
Another case where `__has_builtin` would be nice.

Reply:
You can actually do something like this:

```c++
#if defined(__has_builtin) && __has_builtin(__builtin_foo)
```

This will correctly evaluate the `__has_builtin` when using Clang, and be equivalent to `#if 0` when using a different compiler. The Apple SDK headers do this quite often.